### PR TITLE
librepcb: update to 0.1.7.

### PR DESCRIPTION
**Pull request by ii8**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv6l-musl

### CODE REVIEW

**Comment by abenson** [comment]
> I tested the changes in this PR: NO

Why not?

**Comment by ii8** [comment]
> Why not?

Because I was on a complete potato pc. Tested it now though.

File: srcpkgs/librepcb/template [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-version=0.1.5
-revision=2
-build_style=qmake
-configure_args="-r librepcb.pro"
-hostmakedepends="unzip qt5-qmake qt5-host-tools"
-makedepends="qt5-devel qt5-svg-devel zlib-devel"
+version=0.1.7
+revision=1
+build_style=cmake
+hostmakedepends="unzip cmake qt5-qmake qt5-host-tools"
```
Comment (classabbyamp) [inline srcpkgs/librepcb/template:6]:
```suggestion
hostmakedepends="unzip qt5-qmake qt5-host-tools"
```
not necessary with cmake build_style

File: srcpkgs/librepcb/template [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-version=0.1.5
-revision=2
-build_style=qmake
-configure_args="-r librepcb.pro"
+version=0.1.7
+revision=1
+build_style=cmake
 hostmakedepends="unzip qt5-qmake qt5-host-tools"
-makedepends="qt5-devel qt5-svg-devel zlib-devel"
+makedepends="zlib-devel qt5-devel qt5-tools-devel qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds qt5-svg-devel occt-devel glu-devel openssl-devel"
```
Comment (classabbyamp) [inline srcpkgs/librepcb/template:7]: wrap at ~80 characters

**Pull request closed without merging**